loomi/regulariza-morador-client
# Feature/show error messages login

## Feature
 -  Change texts in landing page
 - add custom error messages to login and registration

**What I did:**
 - delete cards

**How to test:**

1. In your browser go to [stage](http://project.netlify.app/) or [localhost](http://localhost:3000/)
2. Check if the component/view looks just like the [mockup](https://xd.adobe.com/) in different screen sizes (width between 1024px and 1920px)
3. Test the things that you think are worth testing, even the ones that are not in this description

![image](https://user-images.githubusercontent.com/32523401/136054299-d86ff7d5-98b0-4f7d-8194-aa6d3cdb5487.png)
